sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -1266,7 +1266,7 @@
       "put": {
         "summary": "Creates a new job to schedule update twins or device direct methods on an IoT hub at a scheduled time.",
         "description": "Creates a new job to schedule twin updates or direct methods on IoT Hub at a scheduled time. See https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-jobs for more information.",
-        "operationId": "JobClient_CreateJob",
+        "operationId": "Jobs_Create",
```

Comment:
Maybe check with Basel on this one. I would expect this to be "Jobs_CreateScheduledJob" from what I've heard from him about "v1" vs "v2" jobs being essentially "importExportJob" vs "scheduledJob"

Reply:
Maybe the v1 jobs and v2 jobs should even have their own subclients to make this distinction more clear?

Reply:
For instance, this API would become "ScheduledJobs_Create" instead of "Jobs_CreateScheduledJob"

Reply:
I like the idea of v1 vs v2 client. I'll sync with Basel on this.

Reply:
We synched offline. My vote would be on Tim's first comment. That is, a single client, with more specific method names, such as CreateScheduledJob, or CreateImportJob. We can discuss during our next Jobs API design meeting.

Reply:
synced up with Basel, decided to leave all these in single client and just rename operation id to add scheduled job